config/events/events_config_map.yaml

```diff
@@ -92,4 +92,5 @@ data:
       - VPNKeyRotationStart
       - VPNKeyRotationConfigUpdated
       - CertificateJobCreationFailed
-      - CertificatesRenewNow
\ No newline at end of file
+      - CertificatesRenewNow
+      - VPNKeyRotationConfigConfigDeletedForcefully
```

Comment:
Can we add an event for successful delete of VPNKeyRotationConfig by controller

Reply:
Its not done by controller. Its the owner ref.